spk/zerotier/src/dsm-control.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
+DNAME="ZeroTier"
+PACKAGE="zerotier"
+INSTALL_DIR="/usr/local/${PACKAGE}"
+ZT_HOME_DIR="/var/lib/zerotier-one"
+ZEROTIER="${INSTALL_DIR}/bin/zerotier-one"
+WATCHDOG="${INSTALL_DIR}/bin/zerotier-watchdog.sh"
+PID_FILE="${ZT_HOME_DIR}/zerotier-one.pid"
+LOG_FILE="/var/log/zerotier-one.log"
```

Comment:
The same way, logs should go into `/var/packages/zerotier/target/var`. By the way, generic service handles logging for you.